R/regression.R

```diff
@@ -667,12 +667,14 @@ Regression <- function(formula = as.formula(NULL),
         }
```

Comment:
What is this if statement doing?

Reply:
Hard to see since github didnt include the `if` statement in the comment. Given the line reference, I'm guessing its the one looking at partial data.  In the case of partial data, the variables `.estimation.data` and `.weights` are created by a call to `EstimationData` https://github.com/Displayr/flipRegression/blob/fb7b586a60b5d93f63d0ace366957f743407f85f/R/regression.R#L406 (line 407) and this isn't executed if partial data is used. (no missing values to remove). The statement here defines the variables required in the later code.